### PR TITLE
Institution Ratings

### PR DESCRIPTION
**Pull request by devin-mccurdy**
## Description
Institution ratings data is now generated on version build that is returned in the institution search and profile responses.

The ratings data is calculated from the uploaded `school_ratings`

[ZenHub Issue](https://app.zenhub.com/workspaces/vft-59c95ae5fda7577a9b3184f8/issues/department-of-veterans-affairs/va.gov-team/13620)

Include migration PR: https://github.com/department-of-veterans-affairs/gibct-data-service/pull/734

## Testing done
Tested locally and Q/A reviewed

## Screenshots
n/a

## Acceptance criteria
- [x] Ratings data is generated correctly

## Definition of done
- [x] Events are logged appropriately
- [x] Documentation has been updated, if applicable
- [x] A link has been provided to the originating GitHub issue (or connected to it via ZenHub)
- [x] No sensitive information (i.e. PII/credentials/internal URLs/etc.) is captured in logging, hardcoded, or specs

### CODE REVIEW

File: app/models/institution_builder.rb [hunk 1]
```diff
@@ -776,6 +777,92 @@ def self.build_versioned_school_certifying_official(version_id)
...
+        SUM(CASE #{category} WHEN 2 THEN 1 ELSE 0 END),
+        SUM(CASE #{category} WHEN 3 THEN 1 ELSE 0 END),
+        SUM(CASE #{category} WHEN 4 THEN 1 ELSE 0 END),
+        SUM(CASE #{category} WHEN 5 THEN 1 ELSE 0 END),
+        SUM(CASE WHEN #{category} IS NULL THEN 1 ELSE 0 END),
+        (SUM(CASE #{category} WHEN 1 THEN 1 ELSE 0 END)
+         + SUM(CASE #{category} WHEN 2 THEN 2 ELSE 0 END)
+         + SUM(CASE #{category} WHEN 3 THEN 3 ELSE 0 END)
+         + SUM(CASE #{category} WHEN 4 THEN 4 ELSE 0 END)
+         + SUM(CASE #{category} WHEN 5 THEN 5 ELSE 0 END)) / COUNT(institutions.id)::float,
```
Comment (shawkey-daniel) [inline app/models/institution_builder.rb:807]: Could we use the alias' here like SUM(rated1_count + rated2_count * 2 + rated3_count * 3 ...?

**Review by shawkey-daniel — CHANGES_REQUESTED** [review]

File: app/models/institution_builder.rb (hunk 1 above)
Comment (dmccurdybah) [reply app/models/institution_builder.rb:807]: Just tested to see if that'd work and it looks like it isn't defined yet at that point.

Comment (shawkey-daniel) [reply app/models/institution_builder.rb:807]: Makes sense, thanks

**Review by shawkey-daniel — APPROVED** [review]
LGTM

**Pull request closed without merging**